Example/Database/Tests/Integration/FIRAuthTests.m

```diff
@@ -40,7 +43,9 @@ - (void)tearDown {
 
 - (void)testListensAndAuthRaceCondition {
     [FIRDatabase setLoggingEnabled:YES];
-    id<FAuthTokenProvider> tokenProvider = [FAuthTokenProvider authTokenProviderForApp:[FIRApp defaultApp]];
+    id<FAuthTokenProvider> tokenProvider =
+        [FAuthTokenProvider authTokenProviderWithAuthInterop:
+            FIR_COMPONENT(FIRAuthInterop, [FIRApp defaultApp].container)];
```

Comment:
It would be nice to avoid the default app here, since we've seen it cause some issues with flakiness previously. Luckily, the container shouldn't even be needed: there's an auth interop fake [`FIRAuthInteropFake`](https://github.com/firebase/firebase-ios-sdk/blob/master/Example/Shared/FIRAuthInteropFake.h) that simulates Auth functionality - take a look at the [storage examples here](https://github.com/firebase/firebase-ios-sdk/blob/7e5befc30356604a9f324bb74664a970d148f493/Example/Storage/Tests/Unit/FIRStorageTokenAuthorizerTests.m#L35).

Reply:
Done